apps/src/templates/GradientNavCard.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import ProgressButton from './progress/ProgressButton';
+
+const styles = {
+  card: {
+    flex: 1,
```

Comment:
Can we use flex grid with all our our supported browsers?  I know at one point it wasn't supported everywhere; not sure whether that's changed yet...